lib/Target/RISCV/MCTargetDesc/RISCVELFObjectWriter.cpp

```diff
@@ -58,6 +58,10 @@ unsigned RISCVELFObjectWriter::getRelocType(MCContext &Ctx,
     return ELF::R_RISCV_JAL;
   case RISCV::fixup_riscv_branch:
     return ELF::R_RISCV_BRANCH;
+  case RISCV::fixup_riscv_rvc_jump:
```

Comment:
Can you add a test for the relocation (see test/MC/RISCV/relocations.s)?

Reply:
Add Relocation testcases for fixup_riscv_rvc_jump/branch in relocations.s